src/main/java/com/stefanovskyi/gitdeceiver/FakeRepository.java

```diff
@@ -47,10 +47,10 @@ public void addRemote(String remoteURI) {
         }
     }
     public void generateCommits(GitAuthData gitAuthData, FakeRepository fakeRepository,
-                                        LocalDateTime commitDate, int amountOfDays) {
+                                        LocalDateTime commitDate) {
```

Comment:
No need to remove `amountOfDays`, it still should be passed to method.

Reply:
Done